app/views/user_mailer/invitation_to_claim.text.erb

```diff
@@ -1,34 +1,33 @@
 <% content_for :message do %>
-  Hello from the Archive of Our Own!
+  <%= t '.greeting', app_name: ArchiveConfig.APP_NAME %>
 
-  You're receiving this e-mail because an archive has recently been imported by Open Doors (http://opendoors.transformativeworks.org) into the Archive of Our Own (AO3 - <%= root_url %>), and we believe that the following fanworks belong to you. We'd like to give you a chance to claim (or delete/orphan) these works if you want to. And if you don't already have an account under a different e-mail, we'd like to invite you aboard!
+  <%= t '.text.part1', open_doors_link: "http://opendoors.transformativeworks.org", app_name: ArchiveConfig.APP_NAME, app_short_name: ArchiveConfig.APP_SHORT_NAME, app_url: root_url %>
 
-  You can read announcements about recent archive moves at AO3 News (<%= root_url %>admin_posts?tag=18), and find additional information on Open Doors' FAQ page (http://opendoors.transformativeworks.org/faq) or tutorials page (http://opendoors.transformativeworks.org/tutorials). For any questions not answered in the FAQ, tutorials, or this e-mail, please contact Support at <%= root_url %>support.
+  <%= t '.text.part2', news_link: root_url + "admin_posts?tag=18", open_doors_faq_link: "http://opendoors.transformativeworks.org/faq", open_doors_tutorial_link: "http://opendoors.transformativeworks.org/tutorials", support_link: root_url + "support" %>
```

Comment:
You should be able to simplify this by using `support_url` rather than combining the path with the root_url. The same is probably true `admin_posts_url + "?tag=18"`.